tests/test_wave_01.py

```diff
@@ -141,12 +141,12 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 1
     assert len(updated_data["watched"]) is 2
-    
+    assert updated_data["watched"][1] is HORROR_1
```

Comment:
In this case I suggest `assert HORROR_1 in updated_data["watched"]`.  The requirements don't specify in what order the the movie should be added to the "watched" list.